src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala

```diff
@@ -47,7 +51,60 @@ class PodsResourceTest extends MarathonSpec with Matchers with Mockito {
     parsedResponse should not be(None)
     parsedResponse.map(_.as[Pod]) should not be(None) // validate that we DID get back a pod definition
 
-    response.getMetadata().containsKey(PodsResource.DeploymentHeader)
+    response.getMetadata().containsKey(PodsResource.DeploymentHeader) should be(true)
+  }
+
+  test("update a simple single-container pod from docker image w/ shell command") {
+    implicit val podSystem = mock[PodManager]
+    val f = Fixture.create()
+
+    podSystem.update(any, eq(false)).returns(Future.successful(DeploymentPlan.empty))
+
+    val postJson = """
```

Comment:
You could use ```Pod()``` and have a helper: ```implicit def podToString(pod: Pod): String = Json.stringify(Json.toJson(pod))``` then it'd be a bit easier to read (you could also have it convert to bytes or whatever too if you needed, pretty sure ```Json``` has that too